### PR TITLE
Add Task2

### PR DESCRIPTION
**Pull request by surganovahome2021**

### CODE REVIEW

File: Tasks/Kvaskov/Task2/Task2.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+       37:"тридцать семь", 38:"тридцать восемь", 39:"тридцать девять", 40:"сорок", 41:"сорок одна", 42:"сорок две",
+       43:"сорок три", 44:"сорок четыре", 45:"сорок пять"}
+h1 = {1:"час", 21:"час", 2:"часа", 3:"часа",4:"часа", 22:"часа", 23:"часа", 24:"часа", 12:"часов", 5:"часов", 6:"часов",
+      7:"часов", 8:"часов", 9:"часов", 10:"часов",11:"часов", 12:"часов", 13:"часов", 14:"часов", 15:"часов", 16:"часов", 17:"часов",
+      18:"часов", 19:"часов", 20:"часов"} # 1,21 час: 2, 3, 4, 22, 23, 24  часа - родительный падеж, единственное число: 5-20 часов - родительный падеж, множественное число
+
+h2 = {0:"ровно", 30:"половина"} # Один час ровно, половина часа
+h3 = {0:"нулевого", 1:"первого", 2:"второго", 3:"третьего", 4:"четвертого", 5:"пятого", 6:"шестого", 7:"седьмого", 8:"восьмого",
+      9:"девятого", 10:"десятого", 11:"одинадцатого", 12:"двенадцатого", 13:"тринадцатого", 14:"четырнадцатого", 15:"пятнадцатого", 16:"шестнадцатого", 17:"семнадцатого",
+      18:"восемнадцатого", 19:"девятнадцатого", 20:"дватцатого", 21:"двадцатьпервого", 22:"двадцатьвторого",23:"двадцатьтретьего",24:"нулевого"}
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Kvaskov/Task2/Task2.py:21]: a very thorough data structure, but a bit overcomplicated, from the other hand it leads to a simpler solution logic

File: Tasks/Kvaskov/Task2/Task2.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            result = print(h2[30], " " + h3[current_time.hour+1])
+        elif current_time.minute > 30 and current_time.minute < 45:
+            result = print(dt4[current_time.minute], "минут " + h3[current_time.hour+1])
+        elif current_time.minute >= 45:
+            result = print("без", dt2[current_time.minute], "минут "+ h3[current_time.hour+1])
+        else:
+            print("out of range")
+    else:
+        t1 = int(input("Введите часы в формате 00 = "))
+        t2 = int(input("Введите минуты в формате 00 = "))
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Kvaskov/Task2/Task2.py:43]: it should be a single input in the "hh:mm" format

**Review by MikitaTsiarentsyeu — APPROVED** [review]
Good job, your score is 9

File: Tasks/Kvaskov/Task2/Task2.py (hunk 2 above)
Comment (surganovahome2021) [reply Tasks/Kvaskov/Task2/Task2.py:43]: Hi, create single input

**Pull request merged**